calculator/main.py

```diff
@@ -12,9 +12,19 @@ def create_new_calculator(operations=None):
 
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
-    """
-    pass
 
+    """
+    if operations is None:
```

Comment:
Maybe to avoid the code duplication you could change this into something like:
```python
if not operations:
    operations = {}

return {
    'operations' : operations,
    'history': []
}
```